src/tribler-gui/tribler_gui/widgets/tablecontentdelegate.py

```diff
@@ -90,10 +90,13 @@ class CheckClickedMixin:
     def check_clicked(self, event, _, __, index):
         model = index.model()
         data_item = model.data_items[index.row()]
+        column_position = model.column_position.get(self.column_name, -1)
+        attribute_name = model.columns[column_position].dict_key
```

Comment:
It is probably necessary to use `.get` or `in` here

Reply:
`columns` is a tuple. I'll just add a check above...